### DISCUSSION TITLE
LMTP not working after update to version 3.0.4

### DISCUSSION THREAD

**Discussion by sebastien-timoner** [haraka/Haraka#3401] [Q&A] [answered]
Hello,

Since updating from version 3.0.3 to 3.0.4, I have encountered an issue with the LMTP functionality on my server. Everything was working fine before the update, but now LMTP seems to no longer function correctly. I have attached a screenshot of the error I am receiving to better illustrate the situation.

Could you please help me identify and resolve this issue?

Thank you in advance for your assistance.

Best regards,

Log :
[NOTICE] [54DD71A4-CFBC-493A-9E24-3668EBEEAD1B.1] [core] disconnect ip=209.85.210.176 rdns=mail-pf1-f176.google.com helo=mail-pf1-f176.google.com relay=N early=N esmtp=Y tls=Y pipe=N errors=0 txns=1 rcpts=1/0/0 msgs=0/1/0 bytes=3709 lr="451  (54DD71A4-CFBC-493A-9E24-3668EBEEAD1B.1)" time=1.478

**Comment by msimerson** [comment, marked as answer]
This may be fixed by PR #3407
